bin/ci/verify-pr-build.ps1

```diff
@@ -0,0 +1,12 @@
+param($plan)
+
+$env:HAB_ORIGIN = "ci"
+$env:HAB_LICENSE = "accept-no-persist"
+$env:DO_CHECK = "true"
+
+Set-Location "$PSScriptRoot\..\..\"
+
+hab origin key generate $env:HAB_ORIGIN
+hab pkg build $plan
+
+exit $LASTEXITCODE
```

Comment:
nit: do you need a new line at the end of the file or powershell don't care about that?

Reply:
powershell doesn't care and I guess the linter doesn't either.